src/webgl/p5.RenderBuffer.js

```diff
@@ -8,6 +8,28 @@ p5.RenderBuffer = class {
     this.attr = attr; // the name of the vertex attribute
     this._renderer = renderer;
     this.map = map; // optional, a transformation function to apply to src
+    this._namespace = undefined;
```

Comment:
Can you leave some short comments here about the purpose of these properties. I know they're also defined in the setters down below but I find it helpful to view it all in one place.